### PR TITLE
Sync files with template repo

### PR DESCRIPTION
**Pull request by doosuu**
# Description

## Azure DevOps PBI/Task reference

AB#11555

## Checklist

* [ ] Vehicle App can be started with dapr run and is connecting to vehicle data broker
* [ ] Vehicle App can process MQTT messages and call the seat service
* [ ] Vehicle App can be deployed to local K3D and is running
* [ ] Created/updated tests, if necessary. Code Coverage percentage on new code shall be >= 70%.
* [ ] Extended the documentation in Velocitas repo
* [ ] Extended the documentation in README.md

* [ ] Devcontainer can be opened successfully
* [ ] Devcontainer can be opened successfully behind a corporate proxy
* [ ] Devcontainer can be re-built successfully

* [ ] Release workflow is passing

### CODE REVIEW

**Review by MP91 — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+2.
+3.
+4.
+
+## Context (Environment)
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+
+<!--- Provide a general summary of the issue in the Title above -->
+
```
Comment (BjoernAtBosch) [inline .github/ISSUE_TEMPLATE.md:27]: Duplicate?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+## Detailed Description
+<!--- Provide a detailed description of the change or addition you are proposing -->
+
+## Possible Implementation
+<!--- Not obligatory, but suggest an idea for implementing addition or change -->
```
Comment (BjoernAtBosch) [inline .github/ISSUE_TEMPLATE.md:32]: Is that meaningful in bug reports? I mean, one could do proposal how to fix but this is not required in general. Also, the text should be "" and ""

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (doosuu) [reply .github/ISSUE_TEMPLATE.md:27]: Copied from `vehicle-app-python-sdk`. Let's keep the files in sync.

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (doosuu) [reply .github/ISSUE_TEMPLATE.md:32]: Copied from `vehicle-app-python-sdk`. Let's keep the files in sync.

**Pull request merged**